docs/source/support/faq.rst

```diff
@@ -327,9 +327,16 @@ Q: The status of my node on the status monitor seems incorrect?
 Check when last your node made a commitment by running::
```

Comment:
```suggestion
Check when last your node made a commitment by running:
```

Reply:
Actually this is right - it is because of the format of the command below - only one colon is actually shown.